stock_move_partner_info/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Tecnativa - Pedro M. Baeza
+# Copyright 2015 AvanzOsc (http://www.avanzosc.es)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Move Partner Info",
+    "version": "9.0.1.0.0",
+    "author": "OdooMRP team,"
+              "AvanzOSC,"
+              "Tecnativa - Pedro M. Baeza,"
```

Comment:
Please only tecnativa, and you can remove `OdooMRP team`, as that one is only temporary.